File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0;
        for(auto x: nums){
            int dig =0;
            while(x){
                dig++;
                x=x/10;
            }
            if(dig%2==0){
                count++;
            }
        }
        return count;
    }
};

File: find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.java
class Solution {
    public int findNumbers(int[] nums) {
        int count =0;
        for(int x: nums){
            int digCount =0;
            while(x>0){
                x/=10;
                digCount++;
            }
            if(digCount %2 ==0){
                count++;
            }
        }
        return count;
    }
}